modules/sdk-coin-atom/test/unit/transaction.ts

```diff
@@ -109,6 +170,22 @@ describe('Atom Transaction', () => {
       );
       should.equal(Buffer.from(json.signature as any).toString('base64'), testData.TEST_WITHDRAW_REWARDS_TX.signature);
       should.equal(tx.type, TransactionType.StakingWithdraw);
+
+      tx.loadInputsAndOutputs();
+      should.deepEqual(tx.inputs, [
+        {
+          address: testData.TEST_WITHDRAW_REWARDS_TX.delegator,
+          value: 'UNAVAILABLE',
```

Comment:
Any reason why this value is 'UNAVAILABLE' ?

Reply:
Assuming this is similar to the previous comment (re: not optional and therefore defaulting to some text)